cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -526,6 +526,28 @@ bundle agent cfe_internal_refresh_inventory_view
 
 }
 
+bundle agent cfe_internal_refresh_hosts_view
+# @brief Refresh hosts materialized view every 5 minutes
+{
+  meta:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "tags" slist => { "enterprise_maintenance" };
+
+  commands:
+
+    (policy_server|am_policy_hub).enterprise_edition::
+
+      "$(sys.workdir)/httpd/php/bin/php"
+        args => "$(cfe_internal_hub_vars.docroot)/index.php cli_tasks materialized_hosts_view_refresh",
+        comment => "This refreshes the hosts view. If the hosts view is not refreshed then it will contain stale data.",
+        handle  => "mpf_fresh_hosts_view",
+        classes => kept_successful_command,
```

Comment:
I think that we should remove this classes => kept_successful_command constraint.

It causes the command to be interpreted as a promise kept, instead of a promise repaired, but it is actually a change ot the system, so I think we should interpret it so. We have removed this from other promises for similar reasons before.